### ISSUE TITLE
Add editor for integrated chords (add/modify/delete)

### ISSUE THREAD

**Issue by louis-coding** [bloodybowlers/UkeGeeks-ng#19]
The integrated set of "standard" chords is basically static - no chance to add or modify something permanently without diving into the js code.
Admins should have a chance to permanantly modify the standard chord repository.

**Comment by bloodybowlers** [comment]
Yes. To do this it would need a major rewrite I think.
In the meantime, if we can add the missing chords to the JS, it should be pretty solid.

Another alternative if it's an "exotic" chord  : Users can define custom chords thanks to the ChordPro format too (and it's currently working in UkeGeeks).

**Comment by louis-coding** [comment]
Agreed, but still I hate fiddling around in the js - it's always a 2 step process: edit src/js/scriptasaurus/ukeGeeks.definitions.sopranoUkuleleGcea.js, then combine the files - or edit the single and combined files separately. Far from user-friendly...

On the other hand: how much of a difference would it make (e.g. performance-wise) to load the standard chords from a standalone (uncombined) js file? 
Or add the possibility to load such file "on top" like e.g. the settings.json (incl. the possibility to override default chord definitions)? Deleting integrated chords is the least necessary option IMHO

**Comment by bloodybowlers** [comment]
Yes it's not user friendly at all.

I already disabled the minified version and I use the "merged" one. I made a basch script to merge, but it's a pain to do it everytime. It's out of the question for a user to do that anyway.

 Overriding the config file would be a good idea (like the settings.js file).

I should add a **"conf" subfolder** in the **ugsphp** folder, and put : 
- config.php
- settings.js
- and something like custom_chords.js that will add chords (I need to find a way to specify the tuning to override)

It's pretty hacky (everything is in this project), but it should work...
